### PR TITLE
[Tool] Add diagnostics.sql script for troubleshooting

### PR DESCRIPTION
**Pull request by sduzh**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [x] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: tools/diagnostics.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+show variables\G;
+admin show frontend config\G;
+select * from information_schema.loads\G;
+select * from information_schema.routine_load_jobs\G;
+select * from information_schema.be_metrics\G;
+select * from information_schema.be_bvars\G;
+select * from information_schema.be_cloud_native_compactions\G;
+select BE_ID, NAME, VALUE from information_schema.be_configs\G;
+show proc '/compactions'\G;
+select BE_ID, LOG from information_schema.be_logs where LEVEL='W' or LEVEL='E' limit 1000;
```
[the remark this answers is not in the document]
Comment (sduzh) [reply tools/diagnostics.sql]: Done. 

> is the LOG fields contains the full logline including the timestamp?

Yes

File: tools/diagnostics.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
+-- diagnostics.sql
+
+-- This script collects key system information to help diagnose 
+-- and troubleshoot performance issues.
+
+-- This data can help identify system bottlenecks, 
+-- debug problems, and guide optimization efforts.
+
+-- example:
+-- mysql -h127.0.0.1 -P9030 -uroot < diagnostics.sql > diagnostics.out
```
[the remark this answers is not in the document]
Comment (sduzh) [reply tools/diagnostics.sql]: Updated

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: tools/diagnostics.sql [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+show variables\G;
+admin show frontend config\G;
+select * from information_schema.loads\G;
+select * from information_schema.routine_load_jobs\G;
+select * from information_schema.be_metrics\G;
+select * from information_schema.be_bvars\G;
+select * from information_schema.be_cloud_native_compactions\G;
+select BE_ID, NAME, VALUE from information_schema.be_configs\G;
+show proc '/compactions'\G;
+select BE_ID, LOG from (select BE_ID, TIMESTAMP, LOG from information_schema.be_logs where LEVEL='W' or LEVEL='E' order by TIMESTAMP DESC limit 5000) t order by TIMESTAMP ASC;
```
Comment (wyb) [inline tools/diagnostics.sql:23]: better sort by `be_id`

Comment (sduzh) [reply tools/diagnostics.sql:23]: Good suggestion!

Comment (sduzh) [reply tools/diagnostics.sql]: updated

**Review by kevincai — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by wyb — APPROVED** [review]

**Review by Astralidea — APPROVED** [review]

**Pull request merged**